src/content/en/fundamentals/progressive-web-apps/prpl.md

```diff
@@ -0,0 +1,242 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: 
+
+{# wf_updated_on: 2016-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+
+# The PRPL Pattern {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+Caution: PRPL is a pattern we believe makes sense for applications that can be
```

Comment:
I have a couple of quibbles with this paragraph:

* Not sure whether this site has a menu of callouts with defined semantics, but "Caution" seems unnecessarily strong to me. I would think something more on the order of a "Note" would be appropriate.

* Not sure what "applications that can be granularly served" means, really. I think a fundamental underlying premise of PRPL is that any app can be served in a form more granular than one big bundle, and that there are significant theoretical benefits to exploring serving all apps in more granular form. Does this mean that all apps should immediately stop bundling and start serving individual resources? Of course not, but the way this is phrased right now makes it seem as if there is some special subset of apps that are candidates for granular serving, and I don't think that's the message we want to send right off the bat.

Thoughts on how to convey whatever caution we think is appropriate in a less off-putting way?